modules/sfp_spyonweb.py

```diff
@@ -60,8 +60,8 @@ def watchedEvents(self):
 
     # What events this module produces
     def producedEvents(self):
-        return ['CO_HOSTED_SITE', 'INTERNET_NAME', 'AFFILIATE_DOMAIN', 
-                'WEB_ANALYTICS_ID', 'DOMAIN_NAME']
+        return ['CO_HOSTED_SITE', 'INTERNET_NAME', 'AFFILIATE_INTERNET_NAME',
+                'WEB_ANALYTICS_ID', 'DOMAIN_NAME', 'AFFILIATE_DOMAIN_NAME']
```

Comment:
I can't find code generating this event?

Reply:
Fixed